chapter3.adoc

```diff
@@ -290,15 +294,18 @@ configuration for supervisor domains:
   the RDSM to manage QoS controls for supervisor domains.
 
 Details of `Smsdia`, `Smsdedbg`, `Smsdetrc` and `Smqosid` are described in their
-respective sections in this specification.
+respective sections in this specification. For any optional extensions not
+implemented, the corresponding configuration bits in `msdcfg` are to be treated
+as WPRI. Hence if no optional extensions are implemented, this CSR becomes
```

Comment:
are read-only zero? (WPRI would be appropriate for fields that are not presently defined but may be defined in future and thereby guides SW to preserve their values on a write).